src/main/drivers/flash_m25p16.c

```diff
@@ -271,6 +273,8 @@ static void m25p16_eraseSector(flashDevice_t *fdevice, uint32_t address)
     STATIC_DMA_DATA_AUTO uint8_t readStatus[2] = { M25P16_INSTRUCTION_READ_STATUS_REG, 0 };
     STATIC_DMA_DATA_AUTO uint8_t readyStatus[2];
     STATIC_DMA_DATA_AUTO uint8_t writeEnable[] = { M25P16_INSTRUCTION_WRITE_ENABLE };
+
+    enum {READ_STATUS, WRITE_ENABLE, SECTOR_ERASE};
```

Comment:
Defining types inside functions is not used in Betaflight - can these please be moved to the top of the file?

Reply:
Done